config/services/services.json

```diff
@@ -30,11 +30,13 @@
                 }
               ],
               "date_span": true,
-              "draggable_marker": true
+              "draggable_marker": {
+                "title": "Place marker at output location"
+              }
             }
           },
           {
-            "id": "fronts_detection",
```

Comment:
I would suggest to create a separate id for the wps name, like wps_id and leave the id as it was before. This id is more dedicated to the whole service.

Reply:
Now we use the following state variable to store the service ID: state.selectedService. If we add a separate ID for the WPS process in the services.json (wps_id), should we also store this in a separate state variable (state.selectedWpsService) to keep track of this?

Reply:
Yes you need to create a separate state for the wps_id